consensus/src/epoch_manager.rs

```diff
@@ -304,9 +304,25 @@ impl EpochManager {
 
         let mut safety_rules =
             MetricsSafetyRules::new(self.safety_rules_manager.client(), self.storage.clone());
-        safety_rules
-            .perform_initialize()
-            .expect("Unable to initialize SafetyRules");
+        match safety_rules.perform_initialize() {
+            Ok(()) => { /* Expected */ }
+            Err(Error::ValidatorKeyNotFound(error)) => {
+                error!(
+                    error = error,
+                    "Unable to find the validator consensus key during initialize",
+                );
+            }
+            Err(Error::ValidatorNotInSet(error)) => {
```

Comment:
Same questions as above.

Reply:
The plan here is to also issue an alert, so that the operators are aware their nodes are not in the current validator set. Note that currently, if we hit this case in production, all that happens is that we log "not in set" in LSR and set the validator signer to nothing -- things continue otherwise, and the operator is probably none the wiser.

'cc @davidiw

Reply:
I think there's something already in the run book as is, but we never implemented the alerts as we said that the validator would be crash looping, which it won't any more.

Reply:
Yeah, we'll need to update this.